scripts/install

```diff
@@ -98,7 +98,7 @@ ynh_system_user_create --username=$app
 ynh_script_progression --message="Configuring PHP-FPM..." --weight=50
 
 # Create a dedicated php-fpm config
-ynh_add_fpm_config --usage=medium --footprint=high --phpversion=$YNH_PHP_VERSION --package="$extra_php_dependencies"
+ynh_add_fpm_config --usage=medium --footprint=high --package="$extra_php_dependencies"
```

Comment:
Hmm this will not works on stretch. We need to specify the php version here to install it if needed. Same for the upgrade and restore script